bindings/core/src/method/client.rs

```diff
@@ -301,7 +301,7 @@ pub enum ClientMethod {
     //////////////////////////////////////////////////////////////////////
     // Node indexer API
     //////////////////////////////////////////////////////////////////////
-    /// Fetch alias/basic/NFT/foundry output IDs
+    /// Fetch account/basic/NFT/foundry output IDs
```

Comment:
can it do anchor/delegation?

Reply:
Added them

Reply:
there are other comments that might be missing them too in these changes

Reply:
Updated another place, one is left, but there is a TODO since we didn't filter these types out yet

Reply:
Alrighty